copyable.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord", "~>4.1.0"
+  spec.add_dependency "activerecord", "~>4.2"
```

Comment:
If I understand correctly, this will assert that Copyable requires an ActiveRecord in the 4.x series, preferably 4.2 or better. Which sounds like what we're after.

Reply:
yes, i kept it pretty loose; fortunately the api that i'm relying on works as well for 4.2.8 as it does for 4.2.11, so likely the next check will come at the next version bump we do. Might be good to try the gem with future versions and see how well the tests run. as far as i can tell from reading the rails code, we should be good for at least to version 6.0!